financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountupdate/AccountUpdateRequiredViewModel.kt

```diff
@@ -80,6 +86,24 @@ internal class AccountUpdateRequiredViewModel @AssistedInject constructor(
         navigationManager.tryNavigateTo(InstitutionPicker(referrer))
     }
 
+    private fun openBankAuthRepair(
+        institution: FinancialConnectionsInstitution?,
+        authorization: String,
+        referrer: Pane,
+    ) {
+        if (institution != null) {
+            updateLocalManifest {
+                it.copy(activeInstitution = institution)
+            }
+
+            pendingRepairRepository.set(authorization)
+            navigationManager.tryNavigateTo(Destination.BankAuthRepair(referrer))
+        } else {
+            // Fall back to the institution picker
+            navigationManager.tryNavigateTo(InstitutionPicker(referrer))
```

Comment:
is this considered an error? should we log anything?

Reply:
I’ll combing a few `else` branches to address this!